config/default.json

```diff
@@ -77,6 +77,18 @@
     "ENUM_DATA_EXPIRES_IN_MS": 4170000,
     "PARTICIPANT_DATA_EXPIRES_IN_MS": 60000
   },
+  "PROXY_CACHE": {
+    "enabled": true,
+    "type": "redis",
+    "timeout": 5000,
+    "proxyConfig": {
+      "host": "localhost",
+      "port": 6379,
+      "db": 0,
+      "lazyConnect": false
+    }
+  },
```

Comment:
What do you think about having a flat config here? Something like:
```json
  "PROXY_CACHE": {
    "type": "redis",
    "host": "localhost",
    "port": 6379,
    ...
  }
```
Also do we need the `enabled` ? It is a bit confusing if we enable the cache with it or the proxying. I also hesitate if the whole config should be named `PROXY_CACHE` or `PROXY` or something else. 

@geka-evk, it will be good to have your opinion too for the config structure. I think the goal of the library is not exactly caching but more like proxy discovery.

Reply:
I am completely indifferent. The current structure works perfectly.